spyderlib/utils/inspector/conf.py

```diff
@@ -10,7 +10,7 @@
 from sphinx import __version__ as sphinx_version
 
 # Local imports
-from spyderlib.config import CONF
+from spyderlib.config.config import CONF
```

Comment:
This an error, right? It should be

    from spyderlib.config.main import CONF